modules/openapi-generator/src/main/resources/Java/libraries/resttemplate/ApiClient.mustache

```diff
@@ -595,13 +596,41 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
     public String expandPath(String pathTemplate, Map<String, Object> variables) {
         return restTemplate.getUriTemplateHandler().expand(pathTemplate, variables).toString();
     }
+    
+    private String generateQueryUri(MultiValueMap<String, String> queryParams, Map<String, Object> uriParams) {
```

Comment:
Suggestion: adding a docstring to explaining what this function does

Reply:
Done